pkg/config/template.go

```diff
@@ -177,4 +177,11 @@ type configuration struct {
 		AuditCheckInterval time.Duration `mapstructure:"audit_check_interval"`
 		AuditCheckURL      string        `mapstructure:"audit_check_url"`
 	} `mapstructure:"security"`
+	Email struct {
+		SMTPHost string `mapstructure:"smtp_host"`
+		Port     int    `mapstructure:"port"`
+		Username string `mapstructure:"username"`
+		Password string `mapstructure:"password"`
+		Sender   string `mapstructure:"sender"`
```

Comment:
renaming using `mapstructure` ins't needed here

Reply:
It's needed for SMTPHost as it contains an underscore. From my understanding of the docs, it is preferable to add it even in the case it might not be needed.